playbook.yml

```diff
@@ -3,13 +3,12 @@
   pre_tasks:
     - name: install packages for testing under docker
       yum:
-        name: "{{ item }}"
+        name:
+          - selinux-policy
+          - libselinux-python
+          - openssh-server
+          - which
         state: present
-      with_items:
-        - selinux-policy
-        - libselinux-python
-        - openssh-server
-        - which
-
+       
```

Comment:
There is trailing whitespace on this line which fails the `ansible-lint` tests. Other than that, LGTM.